plotter/labels.go

```diff
@@ -115,3 +112,20 @@ func (l *Labels) GlyphBoxes(p *plot.Plot) []plot.GlyphBox {
 	}
 	return bs
 }
+
+// XYLabeller combines the XYer and Labeller types
+type XYLabeller interface {
+	XYer
+	Labeller
+}
+
+// XYLabels holds XY data with labels.
```

Comment:
Can you add a note on how the labels match up with the XYs?
Maybe something like, "The ith label corresponds to the ith XY."?